frontend/src/components/not-found/styles.scss

```diff
@@ -0,0 +1,47 @@
+.card {
```

Comment:
check the whole file
https://getbootstrap.com/docs/5.0/getting-started/introduction/